lib/src/clock-in-out/clock-out/clock_out_middleware.dart

```diff
@@ -0,0 +1,44 @@
+part of 'clock_out.dart';
+
+class ClockOutMiddleware extends AuthenticationMiddleware {
+  const ClockOutMiddleware();
+
+  @override
+  Future<dynamic> call(HttpRequest req, HttpResponse res) async {
```

Comment:
we should check if the user belongs, in any way, to the organization before letting him clock in